### PR TITLE
GF2 4-6

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: epistles-v2/original/05-epistles.adoc [hunk 1]
```diff
@@ -553,6 +554,7 @@ Amen.
 [.signed-section-signature]
 G+++.+++ F.
 
+SSCC
```
Comment (jaredh159) [inline epistles-v2/original/05-epistles.adoc]: whoops, looks like you only half got this line done 😄

Comment (jaredh159) [reply epistles-v2/original/05-epistles.adoc]: Also, above here, line 533 (`Swarthmore, in Lancashire...`) should be a SSCC as well.

Comment (jaredh159) [reply epistles-v2/original/05-epistles.adoc]: Also, line 665 (`Swarthmore, the 17th of the 3d month...`) also needs to be a SSCC. 👍

File: epistles-v2/original/06-epistles.adoc [hunk 2]
```diff
@@ -420,6 +425,7 @@ So with my love in the everlasting seed.
 [.signed-section-signature]
 G+++.+++ F.
 
+[.signed-section-context-close]
 Swarthmore, the 26th of the 7th month, 1678.
 
 [.centered]
```
Comment (jaredh159) [inline epistles-v2/original/06-epistles.adoc:431]: Below here, on line `435` where it says `- To Friends in the County of 0xford`, can you make _two_ quick tweaks?

1) Remove the `-`
2) the `O` is actually a _zero_, can you make it an `O` instead?

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Looks great, just a few tweaks here and there needed.  Thanks!

File: epistles-v2/original/05-epistles.adoc (hunk 1 above)
Comment (maijapapaya) [reply epistles-v2/original/05-epistles.adoc]: I wonder if I thought i had finished this chapter and actually didn't.....sorry!!!

**Review by jaredh159 — APPROVED** [review]

**Comment by jaredh159** [comment]
Changes look perfect, thanks, merged.

**Pull request merged**